wiki/Contests/en.md

```diff
@@ -2,291 +2,420 @@
+| [#3.2 (osu!taiko)](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [\*namirin - closing eyes](https://osu.ppy.sh/beatmaps/artists/18) | ![osu!taiko][osu!taiko] | ![][flag_HK] [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | ![][flag_US] [Jonarwhal](https://osu.ppy.sh/users/3653035) | *N/A* | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |
+| [#3.3](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [Fractal Dreamers - Fortuna Redux](https://osu.ppy.sh/beatmaps/artists/15) | ![osu!][osu!] | ![][flag_FI] [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | ![][flag_MY] [HootOwlStar](https://osu.ppy.sh/users/4341302) | *N/A* | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |
+
+### Other Official Contests
+
+#### A Labour of Love
+
+A Labour of Love is a team-based spanning over multiple artistic disciplines where participating teams are being tasked to remix/rearrange a song that has been widely known throughout the osu! community, create a beatmap out of the remixed song, and design an original storyboard/background video as a visual accompaniment to the said beatmap. As [indicated by this tweet](https://twitter.com/ppy/status/1333362145224843264), the contest is modelled after [THE BMS OF FIGHTERS events](https://www.bmsoffighters.net/) and other similar competitions.
+
+| Contest No. | Remixed Song | ![Gold Crown][GCrown] (Overall Winner) | ![Silver Crown][SCrown] (Overall Runner-Up) | Community Vote Winner | Results |
+| :-: | :-- | :-- | :-- | :-- | :-- |
+| [1](https://osu.ppy.sh/home/news/2020-11-30-a-labour-of-love) | [Kuba Oms - My Love](https://osu.ppy.sh/beatmaps/artists/13) | extremely valid (![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471), ![][flag_FR] [Shinwoir](https://osu.ppy.sh/users/8984574), ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515), ![][flag_FR] [Realazy](https://osu.ppy.sh/users/918297), and ![][flag_GB] [DeviousPanda](https://osu.ppy.sh/users/4966334)) | schoolboy fans (![][flag_RU] [Shadren](https://osu.ppy.sh/users/745494), ![][flag_UA] [wajinshu](https://osu.ppy.sh/users/6339790), ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231), and ![][flag_RU] [Daycore](https://osu.ppy.sh/users/5596337)) | Team Red but Blue (![][flag_CA] [Redside](https://osu.ppy.sh/users/16039046), ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223), ![][flag_CA] [Chompy](https://osu.ppy.sh/users/7427035), ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323), and ![][flag_CA] [Zer0-G](https://osu.ppy.sh/users/12577911)) | [Results](https://osu.ppy.sh/home/news/2021-04-27-results-a-labour-of-love) |
+
+#### Best of/Community Choice
+
+The Best of/Community Choice is a voting contest that is being held annually to determine the most popular beatmaps <!-- https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results --> in the eyes of the osu! community at each given year.
```

Comment:
```suggestion
The *Best of/Community Choice* contest is a voting contest that is held annually to determine the most popular beatmaps in the eyes of the osu! community each year. <!-- https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results --> 
```

Reply:
think i'll keep this one as that comment is there as an indicator that the contest is indeed meant to sort out "the most popular" (as mentioned in the linked article) instead of "the best"